packages/ai/src/ButtonTemplate.tsx

```diff
@@ -12,6 +12,7 @@ export default function ButtonTemplate(this: Button) {
 			_hideArrowButton={this._hideArrowButton}
 			onClick={this._onClick}
 			onArrowClick={this._onArrowClick}
+			accessibilityAttributes={this._computedAccessibilityAttributes}
```

Comment:
And for this to work we might want to split the ai accessibility attributes like this `{ root: {}, arrowButton: {} }`
Depending on the use case the application developer could set `aria-haspoup="menu"` to the root button or to the  arrow button.